### DISCUSSION TITLE
建议deepin-terminal 搜索时支持匹配到的内容全部高亮

### DISCUSSION THREAD

**Discussion by MicroOps-cn** [linuxdeepin/developer-center#2866] [Features Request & Ideas | 特性请求 & 头脑风暴]
如果无法或者很难实现的话，能否将终端文本保存到临时文件并打开于文本编辑器。

**Comment by ArchieMeng** [comment]
> 能否将终端文本保存到临时文件并打开于文本编辑器。

终端文本你可以使用命令行重定向来实现。或者你说的是希望保留整个的终端历史？

> 建议deepin-terminal 搜索时支持匹配到的内容全部高亮

这个可以看看能不能做。
